corehq/apps/export/models/new.py

```diff
@@ -68,8 +72,9 @@ class ExportColumn(DocumentSchema):
     item = SchemaProperty(ExportItem)
     label = StringProperty()
     # Determines whether or not to show the column in the UI Config without clicking advanced
-    show = BooleanProperty(default=False)
+    is_advanced = BooleanProperty(default=False)
```

Comment:
lol, totally missed that there is already a comment there. Probably time for me to go home...